cmd/generate_repo_config/generate_repo_config.go

```diff
@@ -124,8 +131,17 @@ func generateRepoConfig(configDest, configSource string) error {
 	buf.WriteString("\n")
 	buf.Write(destFile.Format())
 	if err := ioutil.WriteFile(configDest, buf.Bytes(), 0666); err != nil {
-		return err
+		return nil, err
+	}
+
+	var macros []string
+	for _, m := range sortedFiles {
+		f, err := filepath.Rel(filepath.Dir(configSource), m.Path)
```

Comment:
This ran into problems testing because the output dir is not set to the repository (workspace) root, instead it is set to a temp file

Reply:
This was actually found to be an issue based on how ListRepositories always prepends the config-source to the macro file path, since it is passed `workspace=config_path` as an argument. Thus, it is still safe to `TrimPrefix` of the config source
See https://github.com/bazelbuild/bazel-gazelle/issues/1068